### PR TITLE
Drop travis, add gh action

### PR DESCRIPTION
**Pull request by wpoely86**
As travis will be shut down in the near future, replace it with a equivalent github action flow.

### CODE REVIEW

File: .github/workflows/unittest.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+name: run python tests
+on: [push, pull_request]
+
+jobs:
+  python_unittests:
+      runs-on: ubuntu-latest
+      strategy:
+        matrix:
+          python: [2.7, 3.6]
```
Comment (boegel) [inline .github/workflows/unittest.yml:9]: Why not test for a couple more Python 3 versions too?

```suggestion
          python: [2.7, 3.6, 3.7, 3.8, 3.9]
```

**Review by boegel — CHANGES_REQUESTED** [review]

File: .github/workflows/unittest.yml (hunk 1 above)
Comment (wpoely86) [reply .github/workflows/unittest.yml:9]: we don't test those anywhere else?

Comment (boegel) [reply .github/workflows/unittest.yml:9]: But it's trivial to test them here, so why not?

**Review by boegel — APPROVED** [review]

**Comment by wpoely86** [comment]
I guess the result will only show up here after merging it. You can see it in my fork: https://github.com/wpoely86/vsc-base/runs/1971252893

**Pull request merged**